app/assets/stylesheets/application.css

```diff
@@ -1,44 +1,140 @@
+h1 {
+  font-family: "Modak", "Courier New", sans-serif;
+  text-align: center;
+  font-size: 55px;
+  font-weight: lighter;
+  margin: 20px;
+}
+
 body {
   width: 960;
-  margin-left 0;
+  margin-left: 0;
   margin-right: 0;
-  font-family: Lato, sans-serif;
+  font-family: "Lato", sans-serif;
+  background-color: rgb(51, 102, 255);
+  font-size: 18px;
+}
 
+.index {
+  background-color: rgba(51, 204, 51, .8);
+  margin: 0 auto;
+  width: 960px;
+  border-style: solid;
+  border-width: 5px;
+  border-color: rgb(255, 51, 204);
+  border-radius: 15px;
+  padding-bottom: 10px;
+  padding-left: 5px;
+  padding-right: 10px;
 }
 
-table {
+.persons_tasks {
```

Comment:
Per CSS naming conventions, we  should use the `-` to separate words rather than the `_`.